core/modules/server/routes/get-tiddlers-json.js

```diff
@@ -31,17 +33,24 @@ exports.handler = function(request,response,state) {
 	var excludeFields = (state.queryParameters.exclude || "text").split(","),
 		titles = state.wiki.filterTiddlers(filter);
 	response.writeHead(200, {"Content-Type": "application/json"});
-	var tiddlers = [];
+	var response_body = [];
 	$tw.utils.each(titles,function(title) {
 		var tiddler = state.wiki.getTiddler(title);
 		if(tiddler) {
 			var tiddlerFields = tiddler.getFieldStrings({exclude: excludeFields});
 			tiddlerFields.revision = state.wiki.getChangeCount(title);
 			tiddlerFields.type = tiddlerFields.type || "text/vnd.tiddlywiki";
-			tiddlers.push(tiddlerFields);
+			response_body.push(tiddlerFields);
+		} else {
+			if(include_non_tiddlers) {
```

Comment:
We would generally use "includeNonTiddlers" in preference to underscores. But with the changes I suggested the variable should be "includeMissing".

Reply:
Would you also prefer the camelCase in the json response?

The only problem I'd see with "includeMissing"is that it sounds a little bit paradox at first sight. How can one include something that is missing? That being said I don't have strong feelings on "includeMissing" vs. "includeNonTiddlers" and since you are definitely the one to judge what's intuitive for the TiddlyWiki community I'll go with whatever you prefer. Just let me know and I'll implement it.

Reply:
> Would you also prefer the camelCase in the json response?

I think the JSON response should also be `includeMissing: true`

> The only problem I'd see with "includeMissing"is that it sounds a little bit paradox at first sight

Yes quite! I think it's consistent, though.